validation/reproschema-shacl.ttl

```diff
@@ -144,8 +149,14 @@ reproschema:ProtocolShape a sh:NodeShape ;
             sh:maxCount 1 ;
             sh:path schema:about ],
 
-        [ sh:nodeKind sh:IRI ;
-            sh:path schema:associatedMedia ],
+        [ sh:or ( [ sh:nodeKind sh:IRI ] [sh:node schema:ImageObject] ) ;
```

Comment:
ah i see, this will allow both. i don't think we can do that easily with jsonld types. in jsonld one thing can have two types, but there is no way to simplify which type.